solarforecastarbiter/reports/templates/html/body.html

```diff
@@ -157,6 +162,29 @@ <h5>{{ cost.name }}</h5>
 </div>
 {% endfor %}
 {% endif %}
+{% set outages = report.outages %}
+{% if outages | length > 0 %}
+<h4 id="outages">Outages</h4>
+<p>
+Forecast submissions that fall within the periods listed below are excluded from analysis.
+</p>
+<table class="table">
+  <thead>
+    <tr>
+      <th>Outage Start</th>
+      <th>Outage End</th>
+    </tr>
+  </thead>
+  <tbody>
+    {% for outage in outages %}
+    <tr>
+      <td>{{ outage['start'] }}</td>
+      <td>{{ outage['end'] }}</td>
+    </tr>
+    {% endfor %}
+  </tbody>
+</table>
```

Comment:
We usually put this in a separate file, right?

Reply:
I added this here because the costs metadata was defined in the same file right above. I think its small/straightforward enough to just live in the `body.html`?